File: lambda/lambda_function.py
from __future__ import print_function

import json
import uuid
import decimal
import os
import boto3

# Get the service resource.
dynamodb = boto3.resource('dynamodb')

# set environment variable
TABLE_NAME = os.environ['TABLE_NAME']


def lambda_handler(event, context):
    table = dynamodb.Table(TABLE_NAME)
    # put item in table
    response = table.get_item(
        Item={
            'first_name': 'srikanth'
        }
    )

    return {
        'statusCode': 200,
    }

File: lambda/lambda-function.js
const AWS = require('aws-sdk');

const dynamo = new AWS.DynamoDB();

exports.handler = async function(event) {
  console.log("request:", JSON.stringify(event, undefined, 2));
  var item = await dynamo.getItem({
    TableName: process.env.TABLE_NAME,
    Key:{
      first_name: {
        S: 'srikanth'
      }
    }
  }).promise();
  return {
    statusCode: 200,
    headers: { "Content-Type": "text/plain" },
    body: JSON.stringify({
      event: event,
      item,
    }),
  };
}